omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestConstants.java

```diff
@@ -201,4 +201,9 @@
 	public static boolean SWAGGER_LOGS_ON = true;
 	
 	public static boolean SWAGGER_LOGS_OFF = false;
+	
+	/**
+	 * Constants used for the Server Log REST Service privilege checking
+	 */
+	public static final String GET_SERVER_LOGS = "Get Server Logs";
```

Comment:
Shouldn't you prefix this with PRIV_ to make it clearer from the name?

Reply:
I have updated the variable name. Thanks @dkayiwa